platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/shadowgraph/ShadowEvent.java

```diff
@@ -158,6 +157,6 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        return EventStrings.toMediumString(getEvent());
+        return getEvent().toString();
```

Comment:
is `getEvent().toString()` equivalent to the old `EventStrings.toMediumString(getEvent())`? If not, I think it's worth adding an assertion against a sample output in a unit test.

Reply:
Print format has been changed with this PR

I think a unit test for string format is a good idea, but I will add it to `GossipEvent`, instead of `ShadowEvent`, for a few reasons
- `ShadowEvent` is never printed anywhere
- `ShadowEvent::toString` simply delegates to `GossipEvent`
- `ShadowEvent` will soon be deleted

Reply:
On Cody's suggestion, not adding a unit test after all